src/internet/podcasts/podcastparser.cpp

```diff
@@ -291,7 +291,13 @@ void PodcastParser::ParseOutline(QXmlStreamReader* reader,
           // Parse the feed and add it to this container
           Podcast podcast;
           podcast.set_description(attributes.value("description").toString());
-          podcast.set_title(attributes.value("text").toString());
+          QString title = attributes.value("text").toString();
```

Comment:
It just saves one call to toString() in one case. Negligible, so I'm happy to change it if that's more in line with the Clementine coding standard.